test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -9,6 +9,15 @@
 import org.junit.Test;
 
 public class UtilsTest {
+    @Test
+    public void isAnyNull() throws Exception {
+        final String NULL = null;
```

Comment:
I like your approach of trying to put magic variables, but the keywords usually dont need a magic variable.

Imaging having `boolean TRUE = true` 😆